graylog2-web-interface/src/components/common/PaginatedList.jsx

```diff
@@ -58,7 +58,9 @@ const PaginatedList = ({
   const numberPages = Math.ceil(totalItems / pageSize);
 
   useEffect(() => {
-    setCurrentPage(activePage);
+    if (activePage > 0) {
```

Comment:
Should this also verify that `activePage` is not larger than the total number of pages?

Reply:
This check is mainly for ensuring that `activePage` is not undefined. So we only also check that it is a positive number. The total number of pages not really relevant for this code.

Before we did not show the selected page on component mount, since activePage was undefined but initialPage was already set. Which lead to unset the initial page.

I personally do not think your suggested check is necessary. But I am open for reasoning!

Reply:
Sounds good to me, just making sure it was considered! Thanks